components/Tabs/Tabs.js

```diff
@@ -49,18 +56,23 @@ class Tabs {
 
   init() {
     // select the first link and tab upon ititialization
+    this.active = this.links[0];
+    this.links[0];
   }
 
   updateActive(newActive) {
     // deselect the old active link
+    this.activeLink.deselect();
     // assign the new active link
+    this.activeLink = newActive;
   }
 
   getTab(data) {
     // use the tab item classname and the data attribute to select the proper tab
+    return this.element.querySelector('.Tabs__item[data-tab="${data}"]'); // `.querySelector` selects the `.Tabs__item` class. Which is a box of text that will appear under each of our 3 buttons. I still don't understand what '[data-tab="${data}"]' means.
   }
 
 }
 
-let tabs = document.querySelectorAll(".Tabs");
-tabs = Array.from(tabs).map(tabs => new Tabs(tabs));
+let tabs = document.querySelectorAll(".Tabs"); // `.querySelectorAll` selects all occurences of the `.Tabs` class. There is only one occurence of that class. That class holds the 3 buttons we want.
+tabs = Array.from(tabs).map(tabs => new Tabs(tabs)); //We are grabbing the `tabs` variable we created on the previous line and converting it into an array using `Array.from`.Then, we are using `.map` to ???????????????????????????????????????????????
```

Comment:
you are grabbing all of the tabs and placing them into an array so that you can call the map function, which iterates over each element in `tabs` and calls the Tabs constructor on each of them one after another to create three new objects of that Tabs class.